storage/mysql/schema.sql

```diff
@@ -107,7 +107,7 @@ CREATE TABLE enrollments (
 
     CHECK (topic != ''),
     CHECK (push_magic != ''),
-    CHECK (LENGTH(token) > 0)
+    CHECK (LENGTH(token_hex) > 0)
```

Comment:
Let's just do it like the others — `CHECK (token_hex != '')` :) This was leftover from a column rename.